ecs_logging/_stdlib.py

```diff
@@ -112,6 +112,24 @@ def __init__(self, stack_trace_limit=None, exclude_fields=()):
         self._exclude_fields = frozenset(exclude_fields)
         self._stack_trace_limit = stack_trace_limit
 
+    def _record_error_type(self, record):
+        # upstream doesn't handle booleans for `exc_info`
+        if not record.exc_info:
+            return None
+        if isinstance(record.exc_info, bool):
+            return sys.exc_info()[0].__name__
```

Comment:
Should we guard against `sys.exc_info()` returning `(None, None, None)`?

Reply:
I refactored this a bit. Should be covered now.